data/pigui/modules/info-view/03-econ-trade.lua

```diff
@@ -217,16 +217,26 @@ local function transfer_hyperfuel_mil(hyperdrive, amt)
 end
 
 local function fuelTransferButton(drive, amt)
-	local icon = amt < 0 and icons.chevron_down or icons.chevron_up
+	local color = ui.theme.buttonColors.default
+	local enabled = true
 
-	if ui.button(ui.get_icon_glyph(icon) .. tostring(math.abs(amt))) then
+	-- Don't allow transferring fuel while the hyperdrive is doing its thing
+	if Game.player:IsHyperspaceActive() or Game.player.flightState == "JUMPING"
+										or Game.player.flightState == "HYPERSPACE" then
```

Comment:
General nitpick - we prefer to avoid aligning operands like this; instead use a single tab of indentation on a following line (or break the condition out into a separate local variable).

Reply:
Fixed, I believe.